ceph-releases/jewel/ubuntu/14.04/daemon/config.k8s.sh

```diff
@@ -1,9 +1,20 @@
 #!/bin/bash
 set -e
 
+# log arguments with timestamp
+function log {
```

Comment:
Once we have a common logging function, we can start to add more features to it. So I don't want to add different levels to this PR. Most of the error cases typically add `ERROR` to the string. But since that isn't always consistent, we'd likely either have to update all the strings to prefix some sort of debug level, or pass an extra argument to the log function.